firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -837,6 +839,34 @@ def pbs_for_set(document_path, document_data, option):
     return write_pbs
 
 
+def canonical_strings(field_paths):
```

Comment:
`canonicalize_field_names`?

Reply:
I tried to keep the naming similar to node.  WDYT?

Reply:
meh. functions should be verbs acting on nouns, not just nouns. nouns are classes.